front/src/modules/timesStops/helpers/utils.ts

```diff
@@ -198,10 +199,10 @@ export function updateRowTimesAndMargin(
  * and updates the number of days since departure.
  */
 export function updateDaySinceDeparture(
-  pathWaypointRows: PathWaypointRow[],
+  pathWaypointRows: TimesStopsInputRow[],
   startTime?: IsoDateTimeString,
   keepFirstIndexArrival?: boolean
```

Comment:
I know it doesn't come from your PR but we could harmonize this with what we decided for `secToHoursString`

```suggestion
  { keepFirstIndexArrival?: boolean }
```

Reply:
Done :)